docs/specs/clients/notify/notify-authentication.md

```diff
@@ -1,133 +1,154 @@
 # Notify Authentication
 
-In this document we will describe the authentication payloads for different methods:
+In this document we will describe the authentication payloads for all [RPC methods](./rpc-methods.md).
 
-- Notify Subscription
-- Notify Subscription Response
-- Notify Message
-- Notify Receipt
-- Notify Subscription Update
-- Notify Subscription Update Response
-- Notify Subscription Delete
-- Notify Subscription Delete Response
-
-All of the above authentication payloads are DID JWTs and share the following claims:
+All of the authentication payloads are DID JWTs and share the following claims:
 
 - act - description of action intent. Must be equal to specific value defined in each payload
 - iat - timestamp when JWT was issued
 - exp - timestamp when JWT must expire. Must be equal to specific TTL defined in each payload, plus the iat value
 - ksu - key server for identity key verification
 
-This is achieved using [Dapp Authentication](./dapp-authentication.md) keys which are exposed by the Notify Server but hosted on the Dapp's domain. Also involving [Identity Keys](../../servers/keys/identity-keys).
+"Signed by: client identity key" indicates payloads sent by clients and are authenticated by the Notify Server with [Identity Keys](../../servers/keys/identity-keys).
+
+"Signed by: dapp identity key" indicates payloads sent by the Notify Server and are authenitcated by clients with [Dapp Authentication](./dapp-authentication.md).
 
-## Expired JWT handling
+"Signed by: Notify Server identity key" indicates payloads sent by the Notify Server and are authenitcated by clients with [Notify Server Authentication](./notify-server-authentication.md).
 
-JWT TTLs must be equal to the message TTLs they are sent on.
+## JWT TTLs & Expired JWT handling
+
+JWT TTLs must be equal to the message TTLs they are sent with as defined in [RPC Methods](./rpc-methods.md).
 
 If a client receives a JWT that is expired, it must ignore it as if it was never received. This is to ensure there are no race conditions between message TTL and JWT TTL.
 
 A non-ideal way to avoid the race condition is for the sender to set the message TTL to slightly less than the JWT TTL, however this must not be done as message TTL is a relative time from when the relay receives the message and not absolute like the JWT expiration is.
 
-## Notify Subscription
+## wc_notifyWatchSubscriptions request
+
+- act - description of action intent. Must be equal to "notify_watch_subscriptions"
+- iss - did:key of an identity key. Enables to resolve attached blockchain account.
+- aud - did:key of an identity key. Enables to resolve associated Dapp domain used.
+- sub - blockchain account that this request is associated with (did:pkh)
+- app - dapp's domain URL
```

Comment:
what's the `app` for?
the "cold started" client does not have an idea of any app
additionally subscriptions in the response will refer to many apps

Reply:
Fixed in 26ec86beb0576c667562e2023c3a4fe839728282